faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -35,6 +42,30 @@ object Decoder {
       }
   }
 
+  implicit object CharDecoder extends Decoder[Char] {
```

Comment:
Do you believe we shouldn't be converting to and from char types? What about byte, short and int?